### PR TITLE
a new backend project

### PR DESCRIPTION
**Pull request by ShirAmit96**

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+var express = require('express');
+var app = express();
```
Comment (ArnonEilat) [inline backend/app.js:2]: Don't use `var`.
Use `const` or `let`.
```suggestion
const app = express();
```

https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var

File: backend/app.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+app.get('/', function (req, res) {
+  res.send('Hello World!');
+});
+app.listen(3000, function () {
```
Comment (ArnonEilat) [inline backend/app.js:6]: Arrow functions is less error prone:
```suggestion
app.listen(3000, () => {
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

File: backend/package.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (ArnonEilat) [inline backend/package.json:7]: There is a convention about npm packages:
Running `npm start` should start a package.

```suggestion
    "start": "node app.js"
```

https://docs.npmjs.com/cli/v8/commands/npm-start

**Review by ArnonEilat — CHANGES_REQUESTED** [review]
Good Job

**Review by ArnonEilat — APPROVED** [review]

**Review by katz-tom — APPROVED** [review]

**Pull request merged**